<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Remote</class>
 <widget class="QMainWindow" name="Remote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="m_pSenderBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
</string>
    </property>
    <property name="text">
     <string>Sender</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pReceiverBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;</string>
    </property>
    <property name="text">
     <string>Receiver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pExitBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 25px;bond;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Remote Desktop</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
